scripts/prune_lm_dir.py

```diff
@@ -425,17 +425,37 @@ def IterateOnce(threshold, step, iter, recovery_step, recovery_size):
                 break
 
     """
-    global logprob_changes, steps, current_num_ngrams
+    global logprob_changes, logprob_changes_for_KL, steps, current_num_ngrams
```

Comment:
these values are used to compute the approximate K-L divergence with -sum(logprob_changes_for_KL)). In case that we go too far and recover from the recovery_step, i think we should overwrite the logprob_change for the dropped steps.